zetaclient/chains/bitcoin/signer/signer.go

```diff
@@ -450,7 +450,7 @@ func (signer *Signer) TryProcessOutbound(
 	if err != nil {
 		logger.Warn().
 			Err(err).
-			Msgf("SignConnectorOnReceive error: nonce %d chain %d", outboundTssNonce, params.ReceiverChainId)
+			Msgf("SignWithdrawTx error: nonce %d chain %d", outboundTssNonce, params.ReceiverChainId)
```

Comment:
- remove chain id
- move nonce to structured log fields

Reply:
https://github.com/zeta-chain/node/pull/3184/commits/97ce52a94a2553fccc9f93e5e7815c3745766281